src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx

```diff
@@ -1,35 +1,167 @@
 ---
-title: Monitor your Capacitor application
+title: Monitor your Ionic Capacitor application
 tags:
   - Mobile monitoring
   - New Relic Mobile Capacitor
   - Get started
-metaDescription: "New Relic's Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+metaDescription: "New Relic's Ionic Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
 redirects:
   - /docs/mobile-monitoring/new-relic-mobile-capacitor/get-started
 ---
 
 import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'
 
-Our New Relic Capacitor agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the Capacitor agent, you'll be able to:
+Our New Relic Ionic Capacitor agent monitors your Ionic Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the agent, you'll be able to:
 
 * **Capture JavaScript errors:** Identify and fix problems quickly.
 * **Track network requests:** See how your app interacts with the backend.
 * **Use distributed tracing:** Drill down into handled exceptions and find the root cause.
 * **Create custom events and metrics:** Understand how your users interact with your app.
 
-For more information, see [New Relic Capacitor agent on Github](https://github.com/newrelic/newrelic-capacitor-plugin).
+For more information, see [New Relic Ionic Capacitor agent on Github](https://github.com/newrelic/newrelic-capacitor-plugin).
 
 ## Compatibility requirements [#requirements]
 
-Before you install the Capacitor agent, ensure your app meets these requirements:
+Before you install the agent, ensure your app meets these requirements:
 
 * [Android-native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
 * [iOS-native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
 
-## Install [#install]
-
-To install the Capacitor agent, follow our [guided install](https://onenr.io/0VRVoxJW1ja), located directly in the UI.
+## Install the agent [#install]
+
+To install the agent, follow our [guided install](https://onenr.io/0VRVoxJW1ja), located directly in the UI.
+
+If you need to manually install for any reason, follow these steps:
```

Comment:
I think we can cut "for any reason" and join the sentence with the previous line.